### PR TITLE
[now-cli] Tackle deploy revamp feedback

### PR DESCRIPTION
**Pull request by lucleray**
- [x] Fix "in your organization"
image

- [x] Add feedback form in `canary`
image

and

- [x] "Detecting framework" -> "Setting up project"
- [x] "Auto-detected<framework> settings" -> "Auto-detected project settings (<framework>)"
- [x] "How would you like to call your new project?" -> "What’s your project’s name?"
- [x] ✅ -> ☑️
- [x] organization -> scope

### CODE REVIEW

File: packages/now-cli/src/util/input/input-project.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ export default async function inputProject(
     const answers = await inquirer.prompt({
       type: 'input',
       name: 'newProjectName',
-      message: `How would you like to call your new project?`,
+      message: `What’s your project’s name?`,
     });
```
Comment (AndyBitz) [inline packages/now-cli/src/util/input/input-project.ts:91]: I wonder if we should default to the directory name or what we find in `now.json` if one exists

**Review by AndyBitz — APPROVED** [review]

File: packages/now-cli/src/util/input/input-project.ts (hunk 1 above)
Comment (styfle) [reply packages/now-cli/src/util/input/input-project.ts:91]: Agreed, it should definitely use `now.json`'s `name` property if it exists. There's no reason to ask.

If there is no `name`, the default should be the current directory name.

File: packages/now-cli/src/commands/dev/dev.ts [hunk 2]
```diff
@@ -17,8 +17,6 @@ export default async function dev(
   args: string[],
   output: Output
 ) {
-  output.dim(`Now CLI ${pkg.version} dev (beta) — https://zeit.co/feedback`);
```
Comment (styfle) [inline packages/now-cli/src/commands/dev/dev.ts:20]: Do we not want feedback for `now dev` anymore?

File: packages/now-cli/src/util/input/input-project.ts (hunk 1 above)
Comment (lucleray) [reply packages/now-cli/src/util/input/input-project.ts:91]: Let's merge that first (it's simply changing the label of the question) and continue this discussion. I think we can do that (have a default name), but there's some case where it's not possible and we need a fallback. I'll open a discussion on Slack 👍

File: packages/now-cli/src/commands/dev/dev.ts (hunk 2 above)
Comment (lucleray) [reply packages/now-cli/src/commands/dev/dev.ts:20]: We moved it here: https://github.com/zeit/now/blob/6a1b1b7173c6f2d3ac2bbc423024b693803d57f2/packages/now-cli/src/index.js#L164-L172

I've updated it to always show the feedback prompt when the command is `dev` (as opposed to only show it for `canary`).

**Pull request merged**